### DISCUSSION TITLE
Discord does not accepting?

### DISCUSSION THREAD

**Discussion by guionardo** [deta/cloud-docs#264] [Help]
In the readme file the discord group redirects to https://discord.com/invite/K3gYNqHXUU

I'm receiving an "Whoops. Unable to accept invite"

Can you help me?

**Comment by zawsq** [comment]
paste in your private discord chat then join  https://discord.gg/bpZKRuXDyx
if it wont work your probably banned

**Reply by guionardo** [reply]
Thanks! I'm in!

**Comment by jnsougata** [comment]
Try this one from browser: https://discord.gg/KXq4TNWYFN

**Reply by guionardo** [reply]
Thanks, but it does not work.
